docs/ProfilerOptions.md

```diff
@@ -0,0 +1,91 @@
+| `--begin function`, `--end function`               | `begin=FUNCTION`, `end=FUNCTION`             | automatically start/stop profiling when the specified native function is executed.                                                                                                                                                                                                                                                                                                                                                                   |
+| `--ttsp`                                           | `ttsp`                                       | time-to-safepoint profiling. An alias for `--begin SafepointSynchronize::begin --end RuntimeService::record_safepoint_synchronized`.</br>It is not a separate event type, but rather a constraint. Whatever event type you choose (e.g. `cpu` or `wall`), the profiler will work as usual, except that only events between the safepoint request and the start of the VM operation will be recorded.                                                 |
+| `--jfropts OPTIONS`                                | `jfropts=OPTIONS`                            | Comma separated list of JFR recording options. Currently, the only available option is `mem` supported on Linux 3.17+. `mem` enables accumulating events in memory instead of flushing synchronously to a file.                                                                                                                                                                                                                                      |
+| `--jfrsync CONFIG`                                 | `jfrsync[=CONFIG]`                           | Start Java Flight Recording with the given configuration synchronously with the profiler. The output .jfr file will include all regular JFR events, except that execution samples will be obtained from async-profiler. This option implies `-o jfr`.</br>`CONFIG` is a predefined JFR profile or a JFR configuration file (.jfc) or a list of JFR events started with `+`.</br></br>Example: `asprof -e cpu --jfrsync profile -f combined.jfr 8983` |
+| `--fdtransfer`                                     | `fdtransfer`                                 | Runs a background process that provides access to perf_events to an unprivileged process. `--fdtransfer` is useful for profiling a process in a container (which lacks access to perf_events) from the host.</br>See [Profiling Java in a container](#https://github.com/async-profiler/async-profiler/blob/master/docs/ProfilingInContainer.md).                                                                                                    |
```

Comment:
Seems like some options are missing. I don't see `filter` and `libpath`.
Check `main/main.cpp`.

Reply:
Addressed in [56300c5](https://github.com/async-profiler/async-profiler/pull/1029/commits/56300c529468865dcf38189dab464e1fff07e352)